### PR TITLE
feat: add a React starter project

### PR DESCRIPTION
**Pull request by reubenberghan**
This is simple React starter project. We have a client who wanted something like this to start building their own custom fields.

### CODE REVIEW

**Comment by art-mywave** [comment]
👍

File: examples/react-starter/src/api.ts
Comment (reubenberghan) [inline examples/react-starter/src/api.ts:1]: Vijay mentioned that all clients will be needing LLM support moving forward (and [it sounds like we're moving towards LLMs anyway](https://mywave.atlassian.net/wiki/spaces/FEW/pages/138379269/Removing+NLP+Business+Overview)) so perhaps we need to consider adding the `llmIntentRecogniseEndpoint` and `llmIntentRecogniseApiKey` as just straight options to `MyWaveUi` or supporting the LLM stuff OOTB with `@mywave/ui-react` somehow 🤔

Comment (art-mywave) [reply examples/react-starter/src/api.ts:1]: we need to have llmIntentRecogniseEndpoint has a stable api, I think it had changed before

Comment (reubenberghan) [reply examples/react-starter/src/api.ts:1]: Yeah that's fair. The more I think about it the more it depends on when the Platform is going to support LLMs OOTB. Once that happens I'm assuming the LLM config will be set up in the BE and we will just have to remove this stuff anyway. Probably need to discuss it with the BE guys

**Pull request merged**